database/migrations/2020_08_18_062421_add_character_alt_boolean.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddCharacterAltBoolean extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('characters', function (Blueprint $table) {
+            $table->boolean('is_alt')->nullable()->after('order');
```

Comment:
Because this is a boolean, we likely want to go with `->default(0)` rather than `->nullable()`;